### PR TITLE
start APD endpoints

### PR DESCRIPTION
**Pull request by brendansudol**
Adds `/apds` GET request; updates relationship between APDs and states

### This pull request is ready to merge when...
- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- ~The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented~
- [x] The change has been documented
  - [x] Associated OpenAPI documentation has been updated

### This feature is done when...
- ~Design has approved the experience~
- ~Product has approved the experience~

### CODE REVIEW

File: api/routes/apds/get.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  UserModel = defaultUserModel
+) => {
+  logger.silly('setting up GET /apds route');
+
+  app.get('/apds', loggedIn, async (req, res) => {
+    logger.silly(req, 'handling GET /apds');
+
+    try {
+      // TODO: we should have a helper method that fetches
+      // and returns state/stateId given userId
```
Comment (mgwalker) [inline api/routes/apds/get.js]: I suppose there's no reason we couldn't attach the user model directly to the request object.  Then if we add a helper method to the user model class, we could have something like `await req.user.model.getState()`?

File: api/routes/apds/get.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      const state = user.related('state');
+      const stateId = state.get('id');
+
+      if (!stateId) {
+        logger.verbose('user does not have an associated state');
+        return res.status(401).end();
+      }
+
+      const whereCondits = { state_id: stateId };
+      const apds = (await ApdModel.where(whereCondits).fetchAll()).toJSON();
```
Comment (mgwalker) [inline api/routes/apds/get.js]: Do we want to restrict this fetch to just specific columns?  I'm guessing there's nothing in our data model that users shouldn't see right now but 🤷‍♂️

File: api/routes/apds/get.js (hunk 1 above)
Comment (brendansudol) [reply api/routes/apds/get.js]: yes, i like that!

File: api/routes/index.js [hunk 3]
```diff
@@ -10,10 +11,11 @@ const openAPI = require('./openAPI');
 module.exports = (
   app,
   activitiesEndpoint = activities,
+  apdsEndpoint = apds,
   rolesEndpoint = roles,
   statesEndpoint = states,
   usersEndpoint = users,
-  formLoggerEndopint = formLogger,
+  formLoggerEndpoint = formLogger,
```
Comment (mgwalker) [inline api/routes/index.js]: What have you got against endo-pints?

File: api/routes/apds/get.js (hunk 2 above)
Comment (brendansudol) [reply api/routes/apds/get.js]: yeah, nothing worried me in this table (at this time), but perhaps we always want to be explicit

**Comment by brendansudol** [comment]
good eye re: endpoint tests -- gonna batch those up at the end!

**Pull request merged**